Sources/XcodeProj/Scheme/XCScheme+TestableReference.swift

```diff
@@ -42,6 +46,7 @@ extension XCScheme {
         func xmlElement() -> AEXMLElement {
             var attributes: [String: String] = ["skipped": skipped.xmlString]
             attributes["parallelizable"] = parallelizable ? parallelizable.xmlString : nil
+            attributes["useTestSelectionWhitelist"] = useTestSelectionWhitelist ? useTestSelectionWhitelist.xmlString : nil
```

Comment:
After making the attribute optional, we should not set it here unless it has a value.

Reply:
I'm not quite sure what to do here.
How is `useTestSelectionWhitelist` different from the `parallelizable` or `testExecutionOrdering` attribute in terms of reading / setting?

Reply:
Sorry. We don’t have to do anything here.